server/utils/forms/inputs/calendarDayInput.test.ts

```diff
@@ -145,4 +145,18 @@ describe(CalendarDayInput, () => {
       })
```

Comment:
I think the commit message here is out of date:

```
in the upcoming `ReportingInput` class
```

— should be `ReportingForm` now?